programs/multisig/src/lib.rs

```diff
@@ -236,6 +236,7 @@ pub struct ExecuteTransaction<'info> {
         seeds = [multisig.key().as_ref()],
         bump = multisig.nonce,
     )]
+    /// CHECK: is_signer will be enabled when it actually gets executed
```

Comment:
Let's change this to ` /// CHECK: multisig_signer is a PDA program signer. Data is never read or written to.` and also move this comment above the attribute, assuming the lint doesn't complain.

Reply:
Hi, thanks for looking. @armaniferrante 

I'm not sure what `move this comment above the attribute` means, I tried to put this above the struct declaration, and over the `#[derive(Accounts)]` attribute but got the error that it wants the `/// CHECK: `

Reply:
I meant above the `#[account(...)]` attribute.

Reply:
got it! done in 77bf80a23823be86b6011a76ef9aef95eb8f3a0f